nevergrad/benchmark/experiments.py

```diff
@@ -1253,7 +1253,7 @@ def far_optimum_es(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
 
 
 @registry.register
-def photonics(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+def photonics(seed: tp.Optional[int] = None, as_tuple: bool=False) -> tp.Iterator[Experiment]:
```

Comment:
```suggestion
def photonics(seed: tp.Optional[int] = None, as_tuple: bool = False) -> tp.Iterator[Experiment]:
```
when there is typing, there are spaces around =